axum-core/src/extract/default_body_limit.rs

```diff
@@ -41,7 +41,7 @@ use tower_layer::Layer;
 ///         post(|request: Request<Body>| async {}),
 ///     )
 ///     .layer(DefaultBodyLimit::max(1024));
-/// # let _: Router<(), _> = app;
+/// # let _: Router<()> = app;
```

Comment:
Does the state type of `Router` have a default actually? Seems like it would be nice in situations like this where you want to annotate a binding type.